.vscode/settings.json

```diff
@@ -33,7 +33,6 @@
 	},
 	// Rust Analyzer config
 	"rust-analyzer.cargo.allTargets": false,
-	"rust-analyzer.check.command": "clippy",
```

Comment:
What was the reason for removing this? Feel free to mention it in #dev-tooling if this was intentionally committed and we can discuss its removal, but I'm currently not aware of any problems that it would cause. I'm going to remove it from this PR for the moment so it can be merged, as I'm assuming it was just inadvertently committed.